bootstrap.sh

```diff
@@ -93,6 +93,11 @@ then
   LIBTOOL_BIN="glibtool"
 else
   LIBTOOL_BIN="libtool"
```

Comment:
When adding more special cases to this `if` statement, we should make them all follow the same pattern:

```
if [ `uname -s 2>/dev/null` = 'Darwin' ]
then
  LIBTOOL_BIN="glibtool"
elif [`lsb_release -si 2>/dev/null` = 'Debian' ]
then
  LIBTOOL_BIN="libtoolize"
else
  LIBTOOL_BIN="libtool"
fi
```